src/config.rs

```diff
@@ -2,44 +2,49 @@ use crate::net::constants::{DEFAULT_MTU, FRAGMENT_SIZE_DEFAULT, MAX_FRAGMENTS_DE
 use std::{default::Default, time::Duration};
 
 #[derive(Clone)]
-/// Struct that contains config values for various aspects of the network
+/// Type that contains the configuration options to configure laminar for special use-cases.
 pub struct Config {
-    /// The maximal amount of time to keep `VirtualConnection`s around before cleaning them up.
+    /// The maximal amount of time to keep connection alive before cleaning them up.
     pub idle_connection_timeout: Duration,
     /// This is the maximal size a packet can get with all its fragments.
     ///
     /// Recommended value: 16384
     pub max_packet_size: usize,
-    /// These are the maximal fragments a packet could be divided into.
+    /// This is the value which specifies the maximal allowed fragments.
```

Comment:
I find it better here again to just say `Value which can specify...`, since we know what this is by looking down :)

Not sure if this is the right pattern but imo I think shorter top descriptions is better.